From: Vishesh Handa <me@vhanda.in>
Date: Wed, 13 Jun 2012 21:26:31 +0000
Subject: Revert "Always rec-connect if forced (ie. when Nepomuk server restarts)"
X-Git-Url: http://quickgit.kde.org/?p=kdelibs.git&amp;a=commitdiff&amp;h=bde5aad1e9f7bfb6f8d5c60197797de4a315158d
---
Revert "Always rec-connect if forced (ie. when Nepomuk server restarts)"

This reverts commit a43824a200ed76cef48a90bd3aee8547698b7c91.

This is done so that 4.8.x works with Soprano 2.7.6
---


--- a/nepomuk/core/nepomukmainmodel.cpp
+++ b/nepomuk/core/nepomukmainmodel.cpp
@@ -94,7 +94,7 @@ public:
 
         // we may get disconnected from the server but we don't want to try
         // to connect every time the model is requested
-        if ( forced || (!m_socketConnectFailed && !localSocketClient.isConnected()) ) {
+        if ( !m_socketConnectFailed && !localSocketClient.isConnected() ) {
             delete localSocketModel;
             localSocketModel = 0;
             QString socketName = KGlobal::dirs()->locateLocal( "socket", "nepomuk-socket" );

From: Vishesh Handa <me@vhanda.in>
Date: Wed, 13 Jun 2012 21:25:32 +0000
Subject: Revert "Disconnect before re-connecting. Soprano does not do that for us."
X-Git-Url: http://quickgit.kde.org/?p=kdelibs.git&amp;a=commitdiff&amp;h=76e0376d7582cfe5b898c56bf3779ca2279dace8
---
Revert "Disconnect before re-connecting. Soprano does not do that for us."

This reverts commit 3708be2d09700b21f934c2129218e1c7ea44efdf.

This is being done so that KDE 4.8.x works with Soprano 2.7.6
---


--- a/nepomuk/core/nepomukmainmodel.cpp
+++ b/nepomuk/core/nepomukmainmodel.cpp
@@ -97,7 +97,6 @@ public:
         if ( forced || (!m_socketConnectFailed && !localSocketClient.isConnected()) ) {
             delete localSocketModel;
             localSocketModel = 0;
-            localSocketClient.disconnect();
             QString socketName = KGlobal::dirs()->locateLocal( "socket", "nepomuk-socket" );
             kDebug() << "Connecting to local socket" << socketName;
             if ( localSocketClient.connect( socketName ) ) {

